site/en/posts/site-refactor-2022.md

```diff
@@ -0,0 +1,92 @@
+---
+title: ChromeOS.dev Refactor Retrospective
+metadesc: We've just completed a major refactor of the ChromeOS.dev codebase. This is why we did it and what we learned along the way.
```

Comment:
The "just" here gives me pause. This implies recency but the post will eventually grow old. Maybe that's not a problem, but I wanted to point it out.

Reply:
Because this isn't meant as long-term documentation, but rather is a point-in-time item with a date attached, I'm kinda OK with the "just"